HHS/OPRE-OPS
# feat: ✨ adds filtering of BLIs not-applicable to approver

## What changed

- adds Agreement Types
- fixes Alert USWDS breaking changes
- adds filtering/disabling of BLIs not-applicable to approver

## Issue

#2849 #2993

## How to test

1. login as admin
2. add a BLI to Agreement 1 with CAN G99PHS9
3. send all 3 BLIs to review
4. login as division director
5. goto approve page
6. see only 2 review cards
7. see table will have Tooltip on BLI from CAN G99PHS9 and will be In_Review
8. approve BLIs
9. will only approve 2 appropriate BLIs
10. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/60a9f918-9f9f-4504-baf4-4ed771c93a6a

## Definition of Done Checklist
